src/clevercloud/CleverStopwords.py

```diff
@@ -23,13 +23,27 @@ def CleverStopwords(words):
     """
     import nltk
     from nltk.corpus import stopwords
+    
+    # Download stopwords from NLTK library
     nltk.download('stopwords')
+    
+    # Test input variable type
     if not isinstance(words, set):
         raise TypeError("Input variable should be a set.")
 
+    # Test data type of values of input variable
+    for item in words:
+        if not isinstance(item, str):
+            raise TypeError("Values of input variable should be a string.")
+
     # NLTK English stopwords
     stopwords = set(stopwords.words("english"))
 
+    try:
+        nltk.download("stopwords")
+    except:
+        raise ImportError("Stopwords are not downloaded from NLTK.")
```

Comment:
Will this part of the code check whether nltk.download("stopwords") exist?

Reply:
It is to check whether the stopwords library is downloaded successfully from NLTK to use as the default stopwords list.

Reply:
Great. Thanks!